visual_behavior/data_access/loading.py

```diff
@@ -975,6 +975,14 @@ def get_extended_stimulus_presentations_for_session(session):
     return extended_stimulus_presentations
 
 
+def get_behavior_model_summary_table():
```

Comment:
Do you use the behavior model results? If not, I'd remove this bit of code, since its just clutter.

Reply:
i dont think so. should be fine to remove it, unless marina or others use it.